content/pl/examples/minikube/server.js

```diff
@@ -0,0 +1,9 @@
+var http = require('http');
+
+var handleRequest = function(request, response) {
+  console.log('Received request for URL: ' + request.url);
```

Comment:
Technically, this line could be translated and in some translations (such as Russian) it is. Should we do it as well?

Reply:
I would not _require_ translating it in this PR @wurbanski because of the precedent that most localizations skip it.

Localizing this would be nice and is friendly to Polish-speaking readers.

Reply:
I'd skip translating it - this is an internal console debug message. Most of debugging output is in English - let's keep it consistent.